x/application/types/utils.go

```diff
@@ -0,0 +1,32 @@
+package types
```

Comment:
I think we should move and rename this file in to an appropriate go pkg. Naming things "utils" is typically a code smell IMHO. Wdyt about something like `poktroll/pkg/crypto/codec.go`?

Reply:
I would review #112 and probably merge it into this repo as it remove this entirely among other changes that seemed worth its own PR